ocaml/typing/persistent_env.ml

```diff
@@ -864,79 +608,48 @@ let report_error ppf =
   function
   | Illegal_renaming(modname, ps_name, filename) -> fprintf ppf
       "Wrong file naming: %a@ contains the compiled interface for@ \
-<<<<<<< HEAD
-       %a when %a was expected"
-      Location.print_filename filename
-      CU.Name.print ps_name
-      CU.Name.print modname
-||||||| 121bedcfd2
-       %s when %s was expected"
-      Location.print_filename filename ps_name modname
-=======
        %a when %a was expected"
       (Style.as_inline_code Location.print_filename) filename
-      Style.inline_code ps_name
-      Style.inline_code modname
->>>>>>> 5.2.0
+      (Style.as_inline_code CU.Name.print) ps_name
+      (Style.as_inline_code CU.Name.print) modname
   | Inconsistent_import(name, source1, source2) -> fprintf ppf
       "@[<hov>The files %a@ and %a@ \
-<<<<<<< HEAD
-              make inconsistent assumptions@ over interface %a@]"
-      Location.print_filename source1 Location.print_filename source2
-      CU.Name.print name
-||||||| 121bedcfd2
-              make inconsistent assumptions@ over interface %s@]"
-      Location.print_filename source1 Location.print_filename source2 name
-=======
               make inconsistent assumptions@ over interface %a@]"
       (Style.as_inline_code Location.print_filename) source1
       (Style.as_inline_code Location.print_filename) source2
-      Style.inline_code name
->>>>>>> 5.2.0
+      (Style.as_inline_code CU.Name.print) name
   | Need_recursive_types(import) ->
       fprintf ppf
-<<<<<<< HEAD
-        "@[<hov>Invalid import of %a, which uses recursive types.@ %s@]"
-        CU.Name.print import
-        "The compilation flag -rectypes is required"
+        "@[<hov>Invalid import of %a, which uses recursive types.@ \
+         The compilation flag %a is required@]"
+        (Style.as_inline_code CU.Name.print) import
+        Style.inline_code "-rectypes"
   | Inconsistent_package_declaration_between_imports (filename, unit1, unit2) ->
       fprintf ppf
         "@[<hov>The file %s@ is imported both as %a@ and as %a.@]"
         filename
-        CU.print unit1
-        CU.print unit2
+        (Style.as_inline_code CU.print) unit1
+        (Style.as_inline_code CU.print) unit2
   | Illegal_import_of_parameter(modname, filename) ->
       fprintf ppf
         "@[<hov>The file %a@ contains the interface of a parameter.@ \
          %a is not declared as a parameter for the current unit (-parameter %a).@]"
         Location.print_filename filename
```

Comment:
Other uses of `Location.print_filename` are now styled as inline code - should we do this one too for consistency?  (And there's another in the next case)

Reply:
Yes, thanks